examples/2d/regularized_inverse_scattering_MNIST.py

```diff
@@ -24,6 +24,7 @@
 import torch.nn as nn
 import torch.optim as optim
 from PIL import Image
```

Comment:
I'm pros incorporating this change and adding "tqdm" to the requirements_optional.txt